src/utils.ts

```diff
@@ -62,3 +66,29 @@ export function getEnvNumber(key: string, defaultValue: number): number {
 
   return numberValue;
 }
+
+export function deduplicate(arr: string[]) {
+  return [...new Set(arr)];
+}
+
+const formatStringSet = (set: Set<string> | string[]) => {
+  return [...set.values()].map(item => `"${item}"`).join(', ');
```

Comment:
`.values()` will fail in node 8, time to deprecate it? <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/values#browser_compatibility>

Reply:
Interesting that the tests don't fail, right? :) The method you're referring to is on an Array. The code uses [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/values#browser_compatibility). So it's all good!


We're deprecating node 8 in 1.0 for different reasons though regardless!